pagopa/pn-frontend-e2e
# PN-9671-e-2-e-refactor-background-login-dei-test-part1

### Issue Description
All tests include a `background` step for login, which is causing issues with log reading. Consequently, there is no trace of the JIRA ID on AWS CodeBuild. Furthermore, several tests are failing for various reasons."

### Changes Made:
- removed `background` fragment from feature tests
- added a unique login step 
- fix failing tests for notification search

### How to Test
Simply run mvn clean test `-Dcucumber.filter.tags=@ricercaNatoficheMittente` and verify if all the described logic has been correctly implemented and tests related to notification search are working properly.
